executor/scripts/build_local.py

```
@@ -125,6 +125,42 @@ def restore_version_source(original_content: str) -> None:
+
+        # Determine binary name based on platform
+        if platform.system() == "Windows":
+            binary_name = "claude.exe"
+        else:
+            binary_name = "claude"
+
+        binary_path = bundled_dir / binary_name
+
+        if binary_path.exists():
+            # PyInstaller destination: claude_agent_sdk/_bundled/
+            dest = f"claude_agent_sdk/_bundled/{binary_name}"
```

Ruff (0.14.14)

[error] Local variable `dest` is assigned to but never used

Remove assignment to unused variable `dest`

(F841)